include/swift/Basic/InitializeLibSwift.h

```diff
@@ -0,0 +1,22 @@
+//===--- InitializeLibSwift.h -----------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef SWIFT_BASIC_INITIALIZELIBSWIFT_H
+#define SWIFT_BASIC_INITIALIZELIBSWIFT_H
+
+extern "C" {
```

Comment:
Can you guard this with __cplusplus?

Reply:
Just to put a ping here that I want to include this as part of my requested changes.